src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -60,8 +61,8 @@
     // abstract, this is fine (as long as no logging has to happen statically in this class).
     protected final Logger logger = LogManager.getLogger(this.getClass());
 
-    @Argument(common=true, optional=true)
-    public List<File> TMP_DIR = new ArrayList<>();
+    @Argument(common=true, optional=true, doc = "List of temp directories to use.")
+    public List<String> TMP_DIR = new ArrayList<>();
```

Comment:
Can you kebabify this argument as part of this PR (`--temp-dir`)? Looks like we missed this one.

Reply:
Actually, we just did a usage check on this argument, and it's ONLY used in `MarkDuplicates`. Given this, @lbergelson and I think that we should just remove this argument completely. Would you be ok with removing it @magicDGS, or are you currently depending on it?

Reply:
I depend on it, but I can move it to the tools that require it.

Reply:
@droazen Do we think a lot of users use this instead `-Djava.io.tmpdir`?  Maybe it's worth keeping a shortcut too?  Sorry to reconsider.

Reply:
I was reconsidering it too (that's why I didn't add a commit), because maybe there are other codepaths using teh java property, which is set here, instead of directly using the parameter. Next commit will not have this fix.

Reply:
But the name was changed on `StandardArgumentDefinitions.TMP_DIR_NAME` after rebase.